lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -214,6 +214,77 @@ speed:
   config_set_append: "speed %s"
   default_value: "auto"
```

Comment:
All the new stp properties per interface are available on n9k, n6k and n7k, so no_exclude is needed for them.